templates/CRM/Case/Form/Activity.tpl

```diff
@@ -268,10 +268,10 @@
   {if $action neq 8 and $action neq 32768 and empty($activityTypeFile)}
   <script type="text/javascript">
     {if $searchRows}
-      cj('#sendcopy').crmAccordionToggle();
+      cj('#sendcopy').prop('open', function(i, val) {return !val;});
```

Comment:
What's the benefit of this?  Are we trying to remove `crmAccordionToggle()`?

Reply:
We discussed that since it's deprecated, it makes sense to use an alternative while we're changing things here. We looked for shorter alternatives but this was the best we could find.

Reply:
Got it.